src/frontend/components/Header/Header.jsx

```diff
@@ -1,15 +1,50 @@
-import { faBars, faClover, faSearch } from "@fortawesome/free-solid-svg-icons";
+import { faBars, faClover } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useEffect } from "react";
+import { useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { NavLink, useNavigate } from "react-router-dom";
+import { Link, NavLink, useLocation, useNavigate } from "react-router-dom";
 
 import { handleLogout } from "../../features/Auth/authSlice";
+import useDetectClickOutside from "../../hooks/useDetectClickOutside";
+import { debounce } from "../../utils/util";
```

Comment:
Suggestion: try using absolute imports to avoid "../../"

Reply:
I would suggest the same. It was giving an error while i was deploying it to Netlify then I had to change the imports

Reply:
Thanks for the suggestions. I will look into absolute imports